docs/user/hosting.rst

```diff
@@ -49,7 +51,13 @@ as we talk about below.
       we offer a CDN as part of our **Pro plan** and above.
       Please contact support@readthedocs.com to discuss how we can enable this for you.
 
-.. _CloudFlare: https://www.cloudflare.com/
+      We bust the cache on the CDN when the following actions happen:
+
+      * Your Project is saved
+      * Your Domain is saved
+      * A new version is built
```

Comment:
Shouldn't this be "A version is (re)built"? It should be possible to build the same version again and have caching invalidated.

```suggestion
      * A version or branch is (re)built
```

Reply:
Agreed. The intention was to make it easy to decode the case of "rebuilding" in order to make a new custom domain appear as the canonical domain -- but our UI doesn't have a button called "Rebuild", it just has "Build", so I think you are :100: